src/components/Alert/Alert.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import theme from './theme';
 
 import { StyledAlert } from './Alert.styled';
 
-const Alert = props => {
-    return (
-        <StyledAlert>{props.children}</StyledAlert>
-    );
-}
+const Alert = (props) => {
+	const { variant } = props;
 
-export default Alert;
\ No newline at end of file
+	return (
+		<ThemeProvider theme={theme}>
```

Comment:
`ThemeProvider` powinien być gdzieś wyżej w hierarchii np. w `Task01` lub nawet w `App` (wtedy do wszystkich elementów masz jednego "dostawcę").
To działa podobnie jak Context API